modules/terraform/errors.go

```diff
@@ -17,7 +17,7 @@ func (err TgInvalidBinary) Error() string {
 type OutputKeyNotFound string
 
 func (err OutputKeyNotFound) Error() string {
-	return fmt.Sprintf("output doesn't contain a value for the key %q", err)
+	return fmt.Sprintf("output doesn't contain a value for the key %q", string(err))
```

Comment:
```suggestion
	return fmt.Sprintf("output doesn't contain a value for the key %s", err)
```
Could we potentially simplify this?

Reply:
So the bug here is that we don't want to call `Error()` inside of `Error()` (recursion). Chaging the `fmt` verb from `%q` to `%s` does nothing to prevent that as that will result in `Error()` being called (as `Error()` is called if a type has it and does not have `String()`. The approach here is exactly the canonical approach [documented in the fmt pkg](https://golang.org/pkg/fmt/):

> To avoid recursion in cases such as
> 
> type X string
> func (x X) String() string { return Sprintf("<%s>", x) }
> 
> convert the value before recurring:
> 
> func (x X) String() string { return Sprintf("<%s>", string(x)) }
>